style.css

```diff
@@ -118,3 +105,523 @@ h2 {
+  line-height: 32px;
+  letter-spacing: 0;
+  text-align: center;
+}
+
+h3 {
+  height: 57px;
+  width: 295px;
+  left: 16px;
+  top: 16px;
+  border-radius: nullpx;
+  font-family: Inter, sans-serif;
+  font-size: 24px;
+  font-weight: 600;
+  line-height: 32px;
+  letter-spacing: 0;
+  text-align: center;
+  padding-top: 130px;
+  color: #344563;
+}
+
+.secondtittle1 {
+  padding-top: 90px;
+}
+
+.background3 {
+  width: 100%;
+  background-color: #ebf0ee;
+  height: 352px;
+}
+
+.java2 {
+  background-color: white;
+  padding: 10px;
+  background-position: 20px 10px, 150px 10px;
+  margin-right: 8px;
+  margin-left: -10px;
+}
+
+.list2 {
+  display: flex;
+  width: 100%;
+  margin-left: -20px;
+  margin-right: -80px;
+  font-family: Inter, sans-serif;
+  font-size: 12px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  text-align: left;
+}
+
+.list3 {
+  display: flex;
+  width: 100%;
+  margin-left: -20px;
+  margin-right: -80px;
+  font-family: Inter, sans-serif;
+  font-size: 12px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  text-align: left;
+}
+
+.java1 {
+  background-color: white;
+  padding: 10px;
+  background-position: 20px 10px, 150px 10px;
+  margin-right: 8px;
+  color: #36b37e;
+}
+
+.ruby1 {
+  background-color: white;
+  padding: 10px;
+  margin-right: 8px;
+}
+
+.html1 {
+  background-color: white;
+  padding: 10px;
+  margin-right: 8px;
+  color: #36b37e;
+}
+
+.css2 {
+  background-color: white;
+  padding: 10px;
+  margin-left: 45px;
+}
+
+.css3 {
+  background-color: white;
+  padding: 10px;
+}
+
+.css4 {
+  background-color: white;
+  padding: 10px;
+  margin-left: 105px;
+}
+
+.mainbutton {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+}
+
+.button {
+  height: 48px;
+  left: 92px;
+  top: 13px;
+  align-items: center;
+  position: relative;
+  border-radius: 4px;
+  padding: 12px;
+  background-color: #36b37f;
+  text-align: center;
+  color: white;
+  font-family: Inter, sans-serif;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+}
+
+.button :hover {
+  background-color: #d0d9d4;
+}
+
+.card {
+  display: grid;
+  grid-auto-flow: row;
+  grid-gap: 30px 30px;
+  font-family: inter, sans-serif;
+  grid-template-rows: repeat(6, 433px);
+  grid-template-columns: repeat(2, 300px);
+  padding-left: 35px;
+  margin-bottom: 100px;
+  border-radius: 1px 0 3px 4px;
+}
+
+.card2 {
+  display: grid;
+  grid-auto-flow: row;
+  grid-gap: -10px -10px;
+  font-family: inter, sans-serif;
+  grid-template-rows: repeat(6, 385px);
+  grid-template-columns: repeat(2, 300px);
+  padding-left: 35px;
+  padding-top: 59px;
+}
+
+.blank {
+  display: flex;
+  background-color: #ebf0ee;
+  border-radius: 10px;
+  height: 210px;
+  width: 100%;
+}
+
+.blank2 {
+  display: flex;
+  background-color: #ebf0ee;
+  height: 352px;
+  width: 100%;
+  border: 1px solid #d0d9d4;
+  border-radius: 10px;
+  margin-bottom: -50px;
+}
```

Comment:
Try adding some `border-radius` to these items too, so they look more alike the Figma design :nerd_face: